apps/frontend/src/components/forms/UpdateAvatarForm/useUpdateAvatarInput.ts

```diff
@@ -0,0 +1,47 @@
+import { usersApi } from 'api-client';
+import { useCallback, useRef, useState } from 'react';
+import { toast } from 'react-toastify';
+
+import { imageSchema } from 'src/validation-schemas/image.schema';
+
+import type { UpdateAvatarInputProps } from './types';
+import type { ChangeEvent } from 'react';
+
+const useUpdateAvatarInput = ({ imageUri: initImage }: UpdateAvatarInputProps) => {
+  const [imageUri, setImageUri] = useState<string>(initImage);
```

Comment:
It is not required to specify the `<string>` type, it can be infered from `initImage`